wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -59,28 +68,36 @@ function add_custom_menu_item ( $items, $args ) {
 }
 add_filter( 'wp_nav_menu_items', 'add_custom_menu_item', 10, 2 );
 
-/* Adds 'Students Archive' page to the menu */
+/** 
+ * Adds 'Students Archive' page to the menu 
+ * */
 function add_students_archive( $items ) {
     $items .= '<li><a href=' . home_url( '/student' ) . '>Students Archive</a></li>';
```

Comment:
A better way to get the archive link from the student CPT is by using `get_post_type_archive_link()`
https://developer.wordpress.org/reference/functions/get_post_type_archive_link/